### ISSUE TITLE
Color schemes are incorrectly installed as themes

### ISSUE THREAD

**Issue by TraceyC77** [sylveon/terminology-themes#14]
# Summary
After installing terminology-themes the "color schemes" are installed as "themes". The description says they are "Color schemes". They show up in Settings - Theme rather than Settings - Colors. This leads to undesirable behavior.

I found this while looking into issue #11 . I was able to install from source after removing the lines referencing tab_shad_*.png from assets/base.edc. Here are the bugs I encountered, which may be fixed by treating the color schemes properly rather than as themes.:

## Cannot select another color scheme from "Colors" if one of the "themes" from this package is set
Selecting one of the installed "themes" prevents me from changing the color scheme through Settings - Colors.

### Actual behavior (bug):
Select one of the color sehemes installed by this repo from Settings - Theme
Go to Settings - Colors and choose any color scheme
The colors do not change

### Expected behavior:
Set the theme to "default"
Go to settings - Colors
Choose any color scheme, see the terminal change to that color set immediately

## When switching between "Colors" And "Theme" in settings, sometimes the "Colors" view does not load properly.
I was unable to reproduce this with the default install of Terminology on a VM. See screenshots.

### Actual behavior (bug):
In settings, when switching from "Theme" to "Colors" sometimes the color list fails to fully load

![terminology_settings_bugged](https://user-images.githubusercontent.com/10402612/147370693-a4542893-7ccb-472a-88bf-d2b37fcf7cdb.png)

### Expected behavior:
The Colors view fully loads.

![terminology_settings_expected](https://user-images.githubusercontent.com/10402612/147370694-ada268c0-02b2-4c60-8bd9-1ecaf2c62849.png)

**Comment by sylveon** [comment]
These themes predates color themes being added in Terminology, and haven't been maintained for a while now because I lost interest in running a Linux desktop. I think nowadays there's a flat theme built-in and then all the "themes" are could be ported to color schemes, since that was the main point for making these in the first place: getting more color schemes within Terminology.

**Comment by simotek** [comment]
Yeah colorschemes are now much simpler and from the latest release you can add them as individual files, there is a pretty good guide now at [[1](https://github.com/borisfaure/terminology/blob/master/COLORSCHEMES.md)], it should be pretty simple to take the [Default.ini](https://github.com/borisfaure/terminology/blob/master/data/colorschemes/Default.ini) and replace them with the colors from the .edc files.

I have done a couple of color schemes for e / terminology themes i'm working on so i'm happy to help but I also don't have time to do the whole lot myself, but if someone does i'll ship them with openSUSE.

1. https://github.com/borisfaure/terminology/blob/master/COLORSCHEMES.md
2. https://github.com/borisfaure/terminology/blob/master/data/colorschemes/Default.ini
